src/pack/stream.ts

```diff
@@ -0,0 +1,46 @@
+import { Readable, Writable } from 'stream'
+
+import all from 'it-all'
+import pipe from 'it-pipe'
+
+import { CarWriter } from '@ipld/car'
+import { importer } from 'ipfs-unixfs-importer'
+import normalizeAddInput from 'ipfs-core-utils/src/files/normalise-input/index'
+import globSource from 'ipfs-utils/src/files/glob-source'
+import { sha256 } from 'multiformats/hashes/sha2'
+
+import { Blockstore } from '../blockstore'
+
+// Node version of toCar with Node Stream Writable
+export async function packToStream ({ input, writable, blockstore }: { input: string | Iterable<string> | AsyncIterable<string>, writable: Writable, blockstore: Blockstore }) {
+  // Consume the source
+  const unixFsEntries = await all(pipe(
```

Comment:
[it-last](https://www.npmjs.com/package/it-last) here would mean we don't have to keep all the unixfs-entries in mem.